hello-world-web/src/main/java/helloworld/HelloWorldServlet.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * (C) Copyright IBM Corporation 2021.
```

Comment:
This exists everywhere, but It says copyright IBM, but the author of the PR is not an IBM employee which means either the intent is to transfer copyright, or this should read differently, probably giving @m-reza-rahman's employer. Unless this is copied from elsewhere where it was IBM Copyright.

Reply:
I changed the copyright to Microsoft. Are we OK otherwise for the merge?